packages/manager/modules/exchange/src/account/account.service.js

```diff
@@ -255,4 +258,15 @@ export default class ExchangeAccount {
       this.PLACEHOLDER_DOMAIN_NAME.toUpperCase()
     );
   }
+
+  /**
+   * Get account rules field.
+   * Can be used to build a country codes list
+   * @returns {Promise} rules: fields rules info
+   */
+  getAccountRules() {
+    return this.$http
+      .post('/newAccount/rules')
+      .then(({ data: rules }) => rules);
```

Comment:
No need to provide subsidiary ?

Reply:
Not required, because i use it only to get a countries phone code.